pkg/internal/venafi/sign.go

```diff
@@ -55,6 +57,10 @@ func (v *Venafi) Sign(csrPEM []byte, duration time.Duration, customFields []inte
 		return nil, err
 	}
 
+	if tmpl.Subject.String() == "" {
+		return nil, ErrorMissingSubject
+	}
```

Comment:
Can we add a unit test for this case, just to ensure that a CSR without Organization/CommonName/any other Subject field fails? Do you know whether Venafi use/accept _any_ value in the Subject block as part of the DN?

Reply:
Yup, it just needs something i there